app/views/record/_record_geo.html.erb

```diff
@@ -37,8 +68,61 @@
       </ul>
     <% end %>
 
-    <% if more_info_geo?(@record) %>
-      <%= render partial: 'more_info_geo', locals: { metadata: @record } %>
+    <!-- We only care about geospatial locations for this, as place names are also subjects. -->
+    <% if @record['locations'].present? %>
+      <h3 class="section-title">Geospatial coordinates</h3>
+      <ul>
+      <% parse_nested_field(@record['locations']).each do |location| %>
```

Comment:
Can you check this logic? I only peeked at a couple records but so far they have not shown coordinates but did include the heading?


https://timdex-ui-pi-gdt-283-re-uyis1j.herokuapp.com/record/gisogm:edu.harvard:6bc7d83d1c27


https://timdex-ui-pi-gdt-283-re-uyis1j.herokuapp.com/record/gismit:US_MA_CAMBRIDGE_P3RR_2010

Reply:
Sorry, should've dropped a note in here. The latest commit fixes this.